GSA/px-bears-drupal
# 178 update content entity configuration

## PR Summary

[1] fix drush cim issue of node.bears_agency.title already exists.
[2] remove content type Bears Life Event. add content type Bears Life Event form.
[3] add initial eligibility length field in benefit

## Related Github Issue

- fixes #178

## Detailed Testing steps

To test in sandbox, 
Check drush cim work well without issue of node.bears_agency.title already exists.
Check content type Bears Life Event removed, Bears Life Event Form added.
Check content type Bears Benefit with a new field Initial Eligibility Length.
